Lesson4.1/src/com/home/model/Computer.java

```diff
@@ -0,0 +1,97 @@
+package com.home.model;
+
+import com.home.service.Reader;
+
+import java.util.Random;
+
+public class Computer {
+
+    private String cpu;
+    private String ram;
+    private String hardDisk;
+    private int lifeCycle;
+
+    public Computer(String cpu, String ram, String hardDisk, int lifeCycle) {
+        this.cpu = cpu;
+        this.ram = ram;
+        this.hardDisk = hardDisk;
+        this.lifeCycle = lifeCycle;
+    }
+
+    public void info() {
+        System.out.println("Model of CPU: " + cpu);
+        System.out.println("Model of RAM: " + ram);
+        System.out.println("Model of hard disk: " + hardDisk);
+        System.out.println("Number of life cycle: " + lifeCycle);
+    }
+
+    public int turnOn(int lifeCycle) {
+        //int cycle = lifeCycle;
+        System.out.println("PC is turning on...");
+        if (getRandomNumber()) {
+            lifeCycle = turnOff(lifeCycle);
```

Comment:
It seems like a wrong behavior that you are calling a turnOff in the turnOn method

Reply:
So methods turnOff and turnOn must run always separately? I thought that when I guess the random number, and pc should turn off, I should run turnOff method.

Reply:
let's see what will happen now when turnOn is called.

you do a random check, if it passes you call turn off (which does another random check) , and that is wrong. You need only one random check per operation to turn on or turn off PC.
Also you don't need to turn off pc on successful turning on